docs/cron/export.sh

```diff
@@ -53,6 +56,17 @@ split_file sentences_detailed.csv
 split_file sentences.csv
 split_file sentences_CC0.csv
 split_file transcriptions.csv
+
+python3 "$SCRIPT_DIR/split_files.py" \
+    user_languages.csv \
+    links.csv \
+    tags.csv \
+    sentences_in_lists.csv \
+    jpn_indices.csv \
```

Comment:
We don't need to split this file. It's a special file that only contains Japanese-English pairs.

Reply:
I added it because a few rows of this file are not classified as Japanese. The script would have enabled to reject them. But this may not be the right place to clean this file...